dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -294,6 +296,24 @@ public Workflow initializeWorkflow(String repositoryId, Workflow workflow) {
         return workflow;
     }
 
+    /**
+     * Initializes a workflow based on GitHub app installation
+     * @param repositoryId Organization and repository (ex. dockstore/dockstore-ui2)
+     * @return stub workflow based on GitHub repository
+     */
+    public Workflow initializeWorkflow(String repositoryId) {
```

Comment:
`initializeWorkflow` can be more descriptive if this is for github app installs only